addons/payment/models/payment_acquirer.py

```diff
@@ -183,10 +183,16 @@ def _inverse_journal_id(self):
             ], limit=1)
             if acquirer.journal_id:
                 if not payment_method:
-                    self.env['account.payment.method.line'].create({
-                        'payment_method_id': acquirer._get_default_payment_method(),
-                        'journal_id': acquirer.journal_id.id,
-                    })
+                    default_payment_method_id = acquirer._get_default_payment_method()
```

Comment:
Maybe take the opportunity to rename this method to make its return value more obvious?
```suggestion
                    default_payment_method_id = acquirer._get_default_payment_method_id()
```

Reply:
Thank you for the review, and that's a good idea so thank you for that too!

Reply:
This is not related to this PR but since you're updating `_inverse_journal_id`, let's discuss it here: I see that you unlink the payment method if you unset the journal, but shouldn't you also unlink the payment method if you just change the journal, say from Journal A to Journal B?

Reply:
The link between the journal and the payment method line is a o2m, so if we change the journal it just changes the journal id on the line to the id of the new journal.

Reply:
Yes but this is a stable version so... you can't modify the method signature...

Reply:
Didn't this feature land in stable right before the internal upgrade? :grin: 
I checked before suggesting the change and can confirm that internal doesn't use this method, so renaming it would have been just fine. This can nonetheless be done in the forward-port to `master`.